docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -44,16 +47,16 @@ The logic behind the choice is as follows:
 @endsphinxdirective
 
 To put it simply, when loading the model to the first device on the list fails, AUTO will try to load it to the next device in line, until one of them succeeds. 
-What is important, **AUTO always starts inference with the CPU**, as it provides very low latency and can start inference with no additional delays. 
-While the CPU is performing inference, AUTO continues to load the model to the device best suited for the purpose and transfers the task to it when ready.
+What is important, **AUTO always starts inference with CPU**, as it provides very low latency and can start inference with no additional delays. 
+While CPU is performing inference, AUTO continues to load the model to the device best suited for the purpose and transfers the task to it when ready.
 This way, the devices which are much slower in compiling models, GPU being the best example, do not impede inference at its initial stages.
-For example, if you use a CPU and a GPU, first-inference latency of AUTO will be better than GPU itself.
+For example, if you use a CPU and a GPU, first-inference latency of AUTO will be better than that of GPU alone.
```

Comment:
```suggestion
For example, if you use a CPU and a GPU, the first-inference latency of AUTO will be better than using GPU alone.
```
Sorry I need to revert this commit. "GPU" is not comparing with "AUTO" in this sentence so we cannot use "that of GPU".